### ISSUE TITLE
For timestamp, command_zsh in parse_history(input_file):

### ISSUE THREAD

**Issue by martisj** [rsalmei/zsh-history-to-fish#4]
I know you have put a lot of love into this @rsalmei . So I gave this tool a shot and got this error:
```
Traceback (most recent call last):
  File "/usr/local/bin/zsh-history-to-fish", line 8, in <module>
    sys.exit(exporter())
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/zsh_history_to_fish/command.py", line 72, in exporter
    for timestamp, command_zsh in parse_history(input_file):
ValueError: not enough values to unpack (expected 2, got 1)
```

Any ideas?

**Comment by rsalmei** [comment]
Hey @martisjm, yes I really have put a lot of love and effort here!
That's why it's odd, it did work last time I checked, but now I've got the same error on my machine.
This tells me ZSH has probably changed something in a newer version, and now for some reason it can include empty lines in its history...

Well, the good news are I just fixed it! It is in #5
And I even made some general command output improvements, and included official support for Python 3.9 and 3.10.
It's already released, just `pip install -U zsh-history-to-fish` and you're good to go!
Please let me know if it's ok now.

**Comment by rsalmei** [comment]
Well, still waiting for your answer, but I'm going to close this one.
